easybuild/easyconfigs/c/ChIP-Seq/ChIP-Seq-1.5-1-goolf-1.4.10.eb

```diff
@@ -15,7 +15,7 @@ description = """The ChIP-Seq software provides methods for the analysis of ChIP
 
 toolchain = {'name': 'goolf', 'version': '1.4.10'}
 
-source_urls = ['http://downloads.sourceforge.net/project/chip-seq/chip-seq/%(version)s/']
+source_urls = ['http://downloads.sourceforge.net/project/chip-seq/chip-seq/1.5.1/']
```

Comment:
@boegel they seem to have changed scheme from e.g. `1.4-2` to e.g. `1.5.3`, but in `1.5.0` and `1.5.1` they used the new scheme in the folder name and the old scheme in the filename...